nextflow.config

```diff
@@ -16,23 +16,25 @@ params {
     genome                     = null
     igenomes_base              = 's3://ngi-igenomes/igenomes/'
     igenomes_ignore            = false
-    fasta                      = null// MultiQC options
```

Comment:
@maxulysse @mirpedrol : It seems that the PR is running the initialisation of that `fasta` param to `null` - why is that?